modules/nixos/nix-build-distributor.nix

```diff
@@ -53,9 +54,9 @@
       hostName = "167.235.38.111";
       sshUser = "builder";
       protocol = "ssh-ng";
-      system = "x86_64-darwin";
+      system = "aarch64-darwin";
```

Comment:
Does this mean it's running on an arm64 machine and the x64-darwin is virtualized?

Reply:
yes, that was the case prior to this.